ppml/base/sgx_default_qcnl.conf

```diff
@@ -0,0 +1,52 @@
+{
+  // *** ATTENTION : This file is in JSON format so the keys are case sensitive. Don't change them.
+  
+  //PCCS server address
+  "pccs_url": "YOUR_PCCS_URL/sgx/certification/v3/",
```

Comment:
user doesn't know the pccs at the base image building time, this need to be passed at attestation runtime